RiotNSE/NotificationService.swift

```diff
@@ -344,7 +347,7 @@ class NotificationService: UNNotificationServiceExtension {
                     msgType = nil
                 }
                 
-                let roomDisplayName = self.store.summary(ofRoom: room.roomId)?.displayname
+                let roomDisplayName = NotificationService.store.summary(ofRoom: room.roomId)?.displayname
```

Comment:
```suggestion
                let roomDisplayName = session.store.summary(ofRoom: room.roomId)?.displayname
```

if we static `store` is no more required.

Reply:
👍